command/agent/config/config_test.go

```diff
@@ -75,6 +75,11 @@ func TestLoadConfigFile_AgentCache(t *testing.T) {
 			ClientCert:    "config_client_cert",
 			ClientKey:     "config_client_key",
 		},
+		UnixSockets: &UnixSockets{
```

Comment:
I wonder if this would make more sense in the actual listener config block?

Reply:
This block will be applicable for all the unix sockets that gets created. Otherwise each listener would have to configure this separately. Another reason was to just mimic what Consul does (which may or may not be the right thing to do).

Reply:
Let me know your thoughts and we can really move it inside if that's the right thing to do.